sem1/2.py

```diff
@@ -0,0 +1,13 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+s=int(input('Введите число журавликов '))
+s1=s/3/2
+s2=(s1+s1)*2
+s3=s/3/2
+if s%3==0 and s1%2==0: 
```

Comment:
Наверное, лучше, сначала проверить входные данные на корректность, а уже потом производить вычисления, чтобы не тратить вычислительные мощности компьютера понапрасну. Условие можно записать проще. Подумайте, на что должно делиться число, если оно делится и на 3 и на 2 одновременно?
```suggestion

if s % 3 == 0 and s1 % 2 == 0:
    s1= s2 = s // 3 // 2
    s2 = (s1 + s1) * 2
```